examples/3dtiles.html

```diff
@@ -44,16 +44,8 @@
                     display: none;
                 }
             }
-            #menuDiv {
-                margin : auto auto;
-                width: 100%;
-                padding: 0;
-                height: 100%;
+            #menuDiv {position: absolute; top:0px; margin-left: 0px;}
```

Comment:
can you reformat this please? (one rule per line, a space after colon)

Reply:
@autra done